### DISCUSSION TITLE
How to use with `rustls` feature `aws-lc-rs`?

### DISCUSSION THREAD

**Discussion by AntoinePrv** [SeaQL/sea-orm#2315] [Q&A] [answered]
Hello,

Quite new to cargo. Is there a way to pass a feature flag to the indirect `rustls` dependency, or does this crate need to add an explicit feature like `sqlx` did?
https://github.com/launchbadge/sqlx/blob/a892ebc6e283f443145f92bbc7fce4ae44547331/Cargo.toml#L82C12-L82C21

**Comment by tyt2y3** [comment, marked as answer]
the best thing you can do is to depend on the specific version of `rustls` directly, and enable the feature flag in your Cargo.toml.

enabled features is global to the cargo workspace, and all crates share the same dependency *only if* their version requirements are compatible.
